app/jinja_filters.py

```diff
@@ -52,6 +52,29 @@ def format_percentage(value):
     return '{}%'.format(value)
 
 
+@blueprint.app_template_filter()
+def format_address_list(address1=None, address2=None):
```

Comment:
I think instead of address1 and address2, it should be something like `user_entered_address` and `metadata_address` maybe?  Just something more descriptive.